src/plugins/action_generate_review_assignment_records/src/generateReviewAssignments.ts

```diff
@@ -61,19 +65,44 @@ module.exports['generateReviewAssignments'] = async function (input: any, DBConn
           isLastLevel: nextReviewLevel === numReviewLevels,
         }
     })
+    console.log('ReviewAssignments', reviewAssignments)
+
+    // Save review_assignment records to database
+    const reviewAssignmentIds = await db.addReviewAssignments(Object.values(reviewAssignments))
 
-    const reviewAssignmentIds = await DBConnect.addReviewAssignments(
-      Object.values(reviewAssignments)
+    // Generate review_assignment_assigner_joins
+    // For now we assume that assigners have no Section restrictions
+    console.log('Generating review_assignment_assigner_join records...')
+    const availableAssigners = await db.getPersonnelForApplicationStageLevel(
+      templateId,
+      stageNumber,
+      nextReviewLevel,
+      'Assign'
+    )
+    const reviewAssignmentAssignerJoins = []
+    for (const reviewAssignmentId of Object.values(reviewAssignmentIds)) {
```

Comment:
Hmm i thought `reviewAssignmentIds` is an array of ids (or possibly an error message) ? Would be good to change even thought Object.values work (also would this break if there was an error message as one of reviewAssignemntIds) ?

![Screen Shot 2021-03-19 at 3 34 34 PM](https://user-images.githubusercontent.com/26194949/111723635-be271080-88c8-11eb-816c-737a1d0f9dda.png)

Reply:
Oh yeah, good spotting. I was getting mixed with `reviewAssignments`, which is an object. Cheers.

6a920982